src/vtktools.jl

```diff
@@ -234,8 +234,9 @@ end
 function calc_node_coordinates(mesh::P4estMesh, nodes, n_visnodes)
   # Extract number of spatial dimensions
   ndims_ = ndims(mesh)
+  ndims_spa = size(mesh.tree_node_coordinates,1)
```

Comment:
Please add a comment explaining why we need this convoluted way and cannot just use `ndims_`.

Also, please be consistent with our formatting conventions (space after comma).

Finally, please do not use abbreviations in variable/type/function names that are not absolutely clear to everyone in the community (unless you *are* really referring to the town in Belgium).

Reply:
Instead of doing this, we can now just extract `NDIMS` and `NDIMS_AMBIENT` (this is the same as the variable `ndims_spa` here) from the type parameters of the mesh.